projects/canopy/src/lib/footer/footer-logo/footer-logo.component.scss

```diff
@@ -1,6 +1,8 @@
 @import '../../../styles/mixins';
 
 .lg-footer-logo {
+  display: contents;
```

Comment:
Is there any alternative to this? I'm guessing not. Only mention it because Safari causes an [a11y issue](https://caniuse.com/css-display-contents) with this.